partiql-ast/src/main/java/org/partiql/ast/v1/expr/ExprRowValue.java

```diff
@@ -0,0 +1,36 @@
+package org.partiql.ast.v1.expr;
+
+import lombok.EqualsAndHashCode;
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.v1.AstNode;
+import org.partiql.ast.v1.AstVisitor;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * TODO docs, equals, hashcode
+ *  Also add optional [ROW] keyword property. https://ronsavage.github.io/SQL/sql-99.bnf.html#row%20value%20constructor
+ */
+@lombok.Builder(builderClassName = "Builder")
+@EqualsAndHashCode(callSuper = false)
+public class ExprRowValue extends Expr {
```

Comment:
(self-review): slight adjustment to `RowValue` modeling to be more consistent with what SQL99 defines -- https://ronsavage.github.io/SQL/sql-99.bnf.html#row%20value%20constructor. In the future, can consider also adding the optional `ROW` keyword.